app/app.js

```diff
@@ -0,0 +1,56 @@
+import {App, IonicApp, Platform} from 'ionic/ionic';
+
+import {HelloIonicPage} from './pages/hello-ionic/hello-ionic';
+import {ListPage} from './pages/list/list';
+
+@App({
+  templateUrl: 'build/app.html'
+})
+class MyApp {
+  constructor(app: IonicApp, platform: Platform) {
+
+    // set up our app
+    this.app = app;
+    this.platform = platform;
+    this.initializeApp();
```

Comment:
The last thing that we do in the constructor is make a call to initializeApp, described below